setup.py

```diff
@@ -36,17 +36,17 @@
             "Programming Language :: Python :: 3.6",
         ],
     install_requires=[
-        'sox>=1.3.3',
+        'sox==1.4.0b0',
```

Comment:
sox 1.4.0 has been released - switch to this instead of beta?

Reply:
Good catch. Will do!